project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clj-http "3.0.0-SNAPSHOT"
+(defproject lambdakazan/clj-http "2.0.4-SNAPSHOT"
```

Comment:
I'm sorry I missed this comment, another PR would work, or you can force-push to your branch and fix these